python-sdk/nuscenes/eval/panoptic/merge_for_panoptic.py

```diff
@@ -0,0 +1,199 @@
+"""
+Script to generate nuScenes-panoptic predictions from nuScenes-tracking and nuScene-lidarseg predictions.
```

Comment:
Can consider changing the file name to `get_panoptic_from_seg_det_or_track.py` or sth similar, to make it informative. Also in line2, add `nuScenes detection` since now merged it to single file.